fsharp/suave/web.fsproj

```diff
@@ -1,16 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>net10.0</TargetFramework>
+    <TargetFramework>net9.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
     <Compile Include="Program.fs" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Update="FSharp.Core" Version="7.0.*" />
-    <PackageReference Include="Suave" Version="2.6.*" />
+    <PackageReference Include="Suave" Version="3.1.0" />
```

Comment:
```suggestion
    <PackageReference Include="Suave" Version="3.1.*" />
```

Reply:
I guess thats ok too; though in general I like to be precise.